streams/src/main/java/org/apache/kafka/streams/TopologyConfig.java

```diff
@@ -117,7 +127,7 @@ public class TopologyConfig extends AbstractConfig {
     public final long cacheSize;
     public final long maxTaskIdleMs;
     public final long taskTimeoutMs;
-    public final String storeType;
+    private final DslStoreSuppliers storeSuppliers;
```

Comment:
why is only this field `private`?

(I suspect the answer is that they should all be private but we just use this class as a loose container object that doesn't bother with getters. Even so, let's just conform to the current pattern and make this public like the rest)